### PR TITLE
Adding descriptions

### PR DESCRIPTION
**Pull request by Iqmaa**
I added article descriptions to the markdown files 'cookies', 'Umbraco security hardening' and 'locking of users and password resets'

### CODE REVIEW

File: 14/umbraco-cms/reference/security/security-hardening.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+---
+description: Learn how to strengthen the security of your Umbraco installation, and reduce the risk of unauthorised access.
```
Comment (eshanrnh) [inline 14/umbraco-cms/reference/security/security-hardening.md:2]:
```suggestion
description: Learn how to strengthen the security of your Umbraco installation, and reduce the risk of unauthorized access.
```

File: 14/umbraco-cms/reference/security/password-reset.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
+---
+description: Learn about the security features put in place to protect Umbraco users from unauthorised access and password breaches.
```
Comment (eshanrnh) [inline 14/umbraco-cms/reference/security/password-reset.md:2]:
```suggestion
description: Learn about the security features put in place to protect Umbraco users from unauthorized access and password breaches.
```

File: 14/umbraco-cms/reference/security/cookies.md [hunk 3]
```diff
@@ -1,3 +1,7 @@
+---
+description: Learn about the cookies required for accessing the umbraco backoffice and their purposes.
```
Comment (eshanrnh) [inline 14/umbraco-cms/reference/security/cookies.md:2]:
```suggestion
description: Learn about the cookies required for accessing the Umbraco Backoffice and their purposes.
```

**Review by eshanrnh — CHANGES_REQUESTED** [review]
Thank you @Iqmaa for the PR 🙌 I've made a few comments—could you take a moment to check them out?

**Review by eshanrnh — APPROVED** [review]
Looks good to me 🙌

**Comment by nul800sebastiaan** [comment]
@Iqmaa Good day! You still have a few days to let us know what you want for Hacktoberfest 🎁, we'd love it if it was an active choice!
If you just want us to plant trees 🌳 then let me know or give the form a quick fill (only 2 fields to fill in for trees).
https://feedback.umbraco.com/umbraco-hacktoberfest-2024/

Thanks!

**Pull request merged**